iina/Extensions.swift

```diff
@@ -330,3 +330,8 @@ extension CharacterSet {
     .union(.urlQueryAllowed)
     .union(.urlFragmentAllowed)
 }
+
+
+extension NSMenuItem {
+  static let dummy = NSMenuItem(title: "Dummy", action: nil, keyEquivalent: "")
```

Comment:
This is a hack IHMO. Just change the method signatures to accept `Any` and pass in `nil`.